Umbraco-Cloud/Troubleshooting/Deployments/Type-Not-Found/index.md

```diff
@@ -0,0 +1,45 @@
+---
+versionFrom: 7.0.0
+---
+
+# Extraction error: "Type not found! "
+
+This issue will present itself as an extraction error on your target environment for a deploy.
+
+![Extraction error on Live](images/Error_Environment.png)
+
+The extraction error happens with two Document Types where one of them have been allowed as a child node type for  the other Document Type.
+
+If the Document Type that have been allowed as the child is then deleted,
+
+the references for the child Document Type is not removed from the parent Document Type.
+
+Because the parent Document Type still have the references from the deleted child Document Type,
+
+the environment will throw an extraction error telling us that it can not find the Document Type that is allowed as a child
+
+when trying to deploy to the next environment.
+
+## How to resolve the extraction error
+
+Open the **More info** to see the details for the error message.
+
+![Extraction error on Live](images/Extraction_error.png)
+
+In the case illustrated above the extraction error is saying:
+
+    "Document type c3bedefc-7eab-4ee2-9941-920ecc9b09b2 not found! This Document type is listed
+    
+    as an allowed child content type of parent Document type Test1 with id: 1087, and guid: 4a06f910-7b1b-4ad6-84db-72481b1ae529"
```

Comment:
How about adding this error message into a code block using ``` ? 🤔  

Here's an example:

```
Document type c3bedefc-7eab-4ee2-9941-920ecc9b09b2 not found! This Document type is listed as an allowed child content type of parent Document type Test1 with id: 1087, and guid: 4a06f910-7b1b-4ad6-84db-72481b1ae529
```

Reply:
That makes total sense! I have just added it and pushed it up  😄